keps/sig-cli/0014-krew.md

```diff
@@ -0,0 +1,237 @@
+---
+kep-number: 14
+title: Krew
+authors:
+  - "@lbb"
+owning-sig: sig-cli
+participating-sigs:
+  - sig-cli
+reviewers:
+  - TBD
+  - "@pwittroc"
+approvers:
+  - "@pwittroc"
+editor: lbb
+creation-date: 2018-07-03
+last-updated: yyyy-mm-dd
+status: provisional
```

Comment:
Given the discussion on the issue it sounds like this KEP is in the `implementable` status. See https://github.com/kubernetes/community/blob/master/keps/0001-kubernetes-enhancement-proposal-process.md#kep-workflow